packages/client/components/ParabolScopingSearchResultsRoot.tsx

```diff
@@ -29,31 +33,42 @@ interface Props {
 }
 
 const ParabolScopingSearchResultsRoot = (props: Props) => {
-  const atmosphere = useAtmosphere()
   const {meeting} = props
   const {teamId, parabolSearchQuery} = meeting
   const {queryString, statusFilters} = parabolSearchQuery as unknown as ParabolSearchQuery
+  const queryRef = useQueryLoaderNow<ParabolScopingSearchResultsRootQuery>(
+    parabolScopingSearchResultsRootQuery,
+    {
+      first: 50,
+      teamIds: [teamId],
+      userIds: [],
+      statusFilters: (statusFilters?.length && statusFilters) || taskScopingStatusFilters,
+      filterQuery: queryString!.trim()
+    }
+  )
   return (
-    <QueryRenderer<ParabolScopingSearchResultsRootQuery>
-      environment={atmosphere}
-      query={query}
-      variables={{
-        first: 50,
-        teamIds: [teamId],
-        userIds: [],
-        statusFilters: (statusFilters?.length && statusFilters) || taskScopingStatusFilters,
-        filterQuery: queryString!.trim()
-      }}
-      fetchPolicy={'store-or-network' as any}
-      render={({props, error}) => {
-        const viewer = (props as any)?.viewer ?? null
-        if (error) return <ErrorComponent error={error} eventId={''} />
-        return <ParabolScopingSearchResults viewer={viewer} meeting={meeting} />
-      }}
-    />
+    <ErrorBoundary fallback={(error) => <ErrorComponent error={error} eventId={''} />}>
+      <Suspense fallback={<MockScopingList />}>
+        {queryRef && <ParabolScopingSearchResultsContainer meeting={meeting} queryRef={queryRef} />}
+      </Suspense>
+    </ErrorBoundary>
   )
 }
 
+interface ParabolScopingSearchResultsContainerProps {
+  queryRef: PreloadedQuery<ParabolScopingSearchResultsRootQuery>
+  meeting: ParabolScopingSearchResultsRoot_meeting
+}
+
+function ParabolScopingSearchResultsContainer(props: ParabolScopingSearchResultsContainerProps) {
+  const {queryRef, meeting} = props
+  const data = usePreloadedQuery<ParabolScopingSearchResultsRootQuery>(query, queryRef, {
+    UNSTABLE_renderPolicy: 'full'
+  })
+  const {viewer} = data
+  return <ParabolScopingSearchResults viewer={viewer} meeting={meeting} />
+}
+
 export default createFragmentContainer(ParabolScopingSearchResultsRoot, {
```

Comment:
`...Container` here, because I'm not going to deal with the `createPaginationContainer` inside the `ParabolScopingSearchResult` component yet, maybe we can consider refactoring it to `usePaginationFragment` in #6283

Reply:
but I did the refactoring to replace createFragmentContainer with `useFragment` for ParabolScopingSearchResultsRoot